M2/Macaulay2/packages/PositivityToricBundles.m2

```diff
@@ -396,6 +397,10 @@ graphToricChernCharacter (ToricVectorBundleKlyachko) := {Verbosity => 0} >> opts
 -- METHOD: separateJets
 ------------------------------------------------------------------------------
 
+-- hT .. hashTable
+-- fun .. function that takes key and value as input and returns true or false
+--selectPairs = (hT, fun) -> hashTable select(pairs hT, (k,v) -> fun(k,v) )
```

Comment:
Is there a reason to re-introduce this `selectPairs` comment?  It was removed when `selectPairs` was added to Core in #3305.